### ISSUE TITLE
Matching dropdown options missing focus/selected highlight

### ISSUE THREAD

**Issue by kirsty-hames** [adaptlearning/adapt-contrib-vanilla#525]
### Subject of the issue
When navigating dropdown options via keyboard up/down arrows, there's no visual focus/highlight.

**Issue introduced during previous PR (Fix 2)**: https://github.com/adaptlearning/adapt-contrib-vanilla/pull/507
**Matching issue for ref:** https://github.com/adaptlearning/adapt-contrib-matching/issues/193

### Steps to reproduce
- _Tab_ to a Matching component, select _Enter_ to expand the dropdown button.
- Use _up_ / _down_ arrows to navigate between the options. 
- Use _spacebar_ to select an option.

### Expected behaviour
The option receiving focus should be highlighted.
The selected option should be highlighted.

### Actual behaviour
No options are highlighted.
